app/src/lib/alive/eventloop-tasks.ts

```diff
@@ -0,0 +1,139 @@
+/* eslint-disable */
```

Comment:
This was a copy&paste, this will come from an npm package in the future, so you can ignore it for now.